framework/hpp_api_vulkan_sample.h

```diff
@@ -317,6 +319,13 @@ class HPPApiVulkanSample : public vkb::HPPVulkanSample
 	 */
 	virtual void prepare_gui();
 
+	/**
+	 * @brief Stores a list of shaders for the active sample, used by plugins to dynamically change the shader
+	 * @param shader_language The shader language for which the shader list will be provided
+	 * @param list_of_shaders The shader list, where paths and shader types are provided
+	 */
+	void store_shader(const vkb::ShaderSourceLanguage &shader_language, const std::vector<std::pair<VkShaderStageFlagBits, std::string>> &list_of_shaders);
```

Comment:
As mentioned above, change `VkShaderStageFlagBits` to `vk::ShaderStageFlagBits`.

Reply:
Done.